helm/frost-server/templates/mqtt-deployment.yaml

```diff
@@ -34,9 +34,9 @@ spec:
           imagePullPolicy: {{ .Values.frost.mqtt.image.pullPolicy | quote }}
           ports:
             - name: mqtt
-              containerPort: 1883
+              containerPort: {{ .Values.frost.mqtt.ports.mqtt.servicePort }}
             - name: websocket
-              containerPort: 9876
```

Comment:
These two I'm pretty sure should not be changed. The `Values.frost.mqtt.ports.XXX.servicePort` setting is the external port for the service, while these variables configure the internal port outside the POD.

Essentially, for each service there are three points where a port is configured:
 - Inside the POD in the FROST code.
 - ouside the POD (the nodePort or containerPort: `Values.frost.XXX.ports.YYY.nodePort`)
 - ouside the service (the serivcePort that the client connects to: `Values.frost.XXX.ports.YYY.servicePort`)

I'm not entirely sure I got the names the right way around :)